taosbenchmark-user-manual-CN.md

```diff
@@ -424,61 +277,80 @@ taosBenchmark 是TDengine进行性能测试的工具应用程序，可以进行T
 }
 ```
 
-### 订阅测试 JSON 参数说明：
+### 订阅测试JSON文件的参数
```

Comment:
according to recent document standards, there should be space between Chinese and English word.